### ISSUE TITLE
Consider adding maplibre as a synonym for mapbox

### ISSUE THREAD

**Issue by louwers** [geostyler/geostyler-cli#400]
The MapLibre project is a FOSS continuation of the source code of Mapbox' client libraries when they got closed-sourced.

The [MapLibre Style Specification](https://maplibre.org/maplibre-style-spec/) is backward compatible with the style specification of Mapbox. Consider adding `maplibre` as a `--source` and `--target`.

**Comment by KaiVolland** [comment]
Yeah that sound like a great idea. We should also [enhance the README](https://github.com/geostyler/geostyler-mapbox-parser/issues/333) of the parser itself to make clear that it is also suitable for maplibre.
